### PR TITLE
update TTS and transport output sample rate to 24000

### PR DESCRIPTION
**Pull request by aconchillo**

### CODE REVIEW

File: src/pipecat/services/assemblyai.py [hunk 1]
```diff
@@ -32,12 +32,12 @@ def __init__(
         self,
         *,
         api_key: str,
-        sample_rate: int = 16000,
+        sample_rate: int = 24000,
```
Comment (markbackman) [inline src/pipecat/services/assemblyai.py:35]: This is an STT service, so we should leave at 16000.

File: src/pipecat/services/assemblyai.py [hunk 2]
```diff
@@ -32,12 +32,12 @@ def __init__(
...
         encoding: AudioEncoding = AudioEncoding("pcm_s16le"),
         language=Language.EN,  # Only English is supported for Realtime
         **kwargs,
     ):
-        super().__init__(**kwargs)
+        super().__init__(sample_rate=sample_rate, **kwargs)
```
Comment (markbackman) [inline src/pipecat/services/assemblyai.py:40]: There's no sample_rate in the STTService class, so we should remove this too.

**Comment by markbackman** [comment]
I ran through a test of all of the services. They sound good!

File: src/pipecat/services/assemblyai.py (hunk 1 above)
Comment (aconchillo) [reply src/pipecat/services/assemblyai.py]: Fixed.

File: src/pipecat/services/assemblyai.py (hunk 2 above)
Comment (aconchillo) [reply src/pipecat/services/assemblyai.py]: Fixed.

**Review by markbackman — APPROVED** [review]
LGTM

**Pull request merged**